Sources/Auth/Core/AuthProvider.swift

```diff
@@ -31,3 +31,17 @@ public struct AuthProvider {
     case credentialsRevoked
   }
 }
+
+public extension AuthProvider.Response {
+  struct Email {
+    let address: String
+    let isPrivate: Bool?
+    let isVerified: Bool?
```

Comment:
Are these properties supposed to be visible to the outside world? If yes, they need to be `public`.

Reply:
Visible yes. Pushed an [update](https://github.com/poviolabs/PovioKit/pull/236/commits/9b3b2f960c6525ba50db2a63f69caac620a7df76).